src/services/index.js

```diff
@@ -1,47 +1,55 @@
-import api from '../utils/api';
+import api from "../utils/api";
 const baseUrl = process.env.REACT_APP_BASE_URL;
+// v3
+const baseURL = process.env.REACT_APP_BASE_URL_V3;
```

Comment:
use REACT_APP_BASE_URL not REACT_APP_BASE_URL_V3 (avoid changing env variable name, this is what is used in deploy)